### PR TITLE
Fix: Shell scripts to pass shellcheck linter for sensor.sh and central.sh

### PR DESCRIPTION
**Pull request by rachana-dani**
## Description
-----------
Fix shell scripts to pass Shellcheck linter for 
```deploy/k8s/sensor.sh```
```deploy/k8s/central.sh```
```deploy/openshift/central.sh```

## Checklist
- [x] Investigated and inspected CI test results
- [ ] Unit test and regression tests added [Does not apply]
- [ ] Evaluated and added CHANGELOG entry if required [Does not apply]
- [ ] Determined and documented upgrade steps [Does not apply]
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs] [Does not apply](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed
CI pipeline is a sufficient indicator but tested by running ```make shell-style```. It passed successfully.

### CODE REVIEW

**Comment by janisz** [comment]
@rachana-dani Style check still failing

File: deploy/k8s/sensor.sh [hunk 1]
```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env bash
+# shellcheck disable=SC1091
+# shellcheck disable=SC2086
```
Comment (janisz) [inline deploy/k8s/sensor.sh:3]: Where do we need this?

Comment (rachana-dani) [reply deploy/k8s/sensor.sh:3]: Its needed for ```ROX_ADMIN_PASSWORD="$(cat ${K8S_DIR}/central-deploy/password)"```. I will move the suppress comment to appropriate position.

**Comment by rachana-dani** [comment]
> @rachana-dani: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> 
> Test name	Commit	Details	Required	Rerun command
> ci/prow/grouped-static-checks	[d46ee71](https://github.com/stackrox/stackrox/commit/d46ee71c1f2027b1d0cf2a75d05d679cc69cd693)	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/stackrox_stackrox/3578/pull-ci-stackrox-stackrox-master-grouped-static-checks/1585680807647776768)	true	`/test grouped-static-checks`
> [Full PR test history](https://prow.ci.openshift.org/pr-history?org=stackrox&repo=stackrox&pr=3578). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:rachana-dani).

Hey @janisz, I have excluded the file names from stylecheck_skip.txt and the grouped_static_checks are failing. This check is to see if files present in stylecheck_skip.txt don't have linting issues anymore. If none of the other tests are failing , why is this one failing? 
Am I missing something? Can you please help me with this?

**Comment by rachana-dani** [comment]
> @rachana-dani: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> 
> Test name	Commit	Details	Required	Rerun command
> ci/prow/grouped-static-checks	[2202929](https://github.com/stackrox/stackrox/commit/2202929ae571856a398655b8916e96fdec81e567)	[link](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/stackrox_stackrox/3578/pull-ci-stackrox-stackrox-master-grouped-static-checks/1585929809010102272)	true	`/test grouped-static-checks`
> [Full PR test history](https://prow.ci.openshift.org/pr-history?org=stackrox&repo=stackrox&pr=3578). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:rachana-dani).

> link

Hey @janisz , I have not made any changes to ```deploy/openshift/sensor.sh``` but yet somehow grouped_static_checks test is failing for me.
Please confirm if this is an issue? I don't think this is an expected behaviour?

**Comment by janisz** [comment]
@rachana-dani Could you rebase on `master`?

**Comment by janisz** [comment]
Please remove `deploy/openshift/sensor.sh` from the list as it's symlink to `k8s/sensor.sh`

`deploy/openshift/sensor.sh -> ../k8s/sensor.sh`

File: scripts/style/shellcheck_skip.txt [hunk 2]
```diff
@@ -2,11 +2,8 @@
...
 deploy/common/docker-auth.sh
 deploy/common/k8sbased.sh
-deploy/k8s/central.sh
 deploy/k8s/deploy-local.sh
 deploy/k8s/deploy.sh
-deploy/k8s/sensor.sh
-deploy/openshift/central.sh
 deploy/openshift/deploy-local.sh
 deploy/openshift/deploy.sh
 deploy/openshift/sensor.sh
```
Comment (janisz) [inline scripts/style/shellcheck_skip.txt:9]:
```suggestion
```

**Comment by rachana-dani** [comment]
Thanks @janisz. @gavin-stackrox @tommartensen, can you please review this PR?

**Comment by gavin-stackrox** [comment]
> Thanks @janisz. @gavin-stackrox @tommartensen, can you please review this PR?

Hi @rachana-dani, thanks for the fix! It looks like @janisz has approved the PR and a single approver is sufficient to merge. Are you able to merge it?

**Pull request merged**